### ISSUE TITLE
Argument labels appeared more than once

### ISSUE THREAD

**Issue by khia** [dreadl0ck/zeus#9]
The following definition fails while I think it shouldn't.
```
        install.ruby:
            description: Install ruby
            help: Install ruby
            arguments:
                - location:String
                - version:String
                - rvm_version:String
```
```
$ bin/zeus install.ruby location=`pwd`/tools/ruby_test version=foo rvm_version=1.29.2
[Jul 20 07:40:17] ERROR handleArgs: failed to execute install.ruby error=argument label appeared more than once: version
```

**Comment by dreadl0ck** [comment]
Interesting. Thanks, I'll have a look.

**Comment by dreadl0ck** [comment]
fixed with commit 0a13d8c, please pull master and rebuild.
